modules/stanford_profile_styles/lib/scss/paragraph-types/react_paragraphs/_multirow.scss

```diff
@@ -0,0 +1,144 @@
+@charset 'UTF-8';
+
+// For full width pages
+.jumpstart-ui--one-column {
+  .react-paragraphs-row {
```

Comment:
I remove the extra selector for the field name so we can use this on other or new fields.

Reply:
Also less specificity.